views/styles.html

```diff
@@ -340,5 +340,16 @@ <h2 class="_style-guide--title">Tabular Data</h2>
     {{table(tableData)}}
   </section>
 
+  <section class="_style-guide--group">
+    <h2 class="_style-guide--title">All Content Types</h2>
+    {% from "components/card.html" import card %}
+    <div class="card--group">
```

Comment:
We need a sample with only a single card, and we need a separate template (`views/layouts/card`) that is for the layout. They should be displayed separately in the style guide.